--- Cheetah-1.0/src/NameMapper.py.orig	2008-06-30 11:05:45.000000000 +0000
+++ Cheetah-1.0/src/NameMapper.py	2008-06-30 11:06:17.000000000 +0000
@@ -139,11 +139,13 @@ Version: $Revision$
 Start Date: 2001/04/03
 Last Revision Date: $Date$
 """
+from __future__ import generators
+
 __author__ = "Tavis Rudd <tavis@damnsimple.com>," +\
              "\nChuck Esterbrook <echuck@mindspring.com>"
 __revision__ = "$Revision$"[11:-2]
 
-from __future__ import generators
+
 import types
 from types import StringType, InstanceType, ClassType, TypeType
 from pprint import pformat
--- Cheetah-1.0/src/Tests/NameMapper.py.orig	2008-06-30 11:17:25.000000000 +0000
+++ Cheetah-1.0/src/Tests/NameMapper.py	2008-06-30 11:17:34.000000000 +0000
@@ -9,10 +9,11 @@ Version: $Revision$
 Start Date: 2001/10/01
 Last Revision Date: $Date$
 """
+from __future__ import generators
+
 __author__ = "Tavis Rudd <tavis@damnsimple.com>"
 __revision__ = "$Revision$"[11:-2]
 
-from __future__ import generators
 import sys
 import types
 import os
