diff -Naur Cheetah-1.0.orig/SetupTools.py Cheetah-1.0/SetupTools.py
--- Cheetah-1.0.orig/SetupTools.py	2006-04-16 21:09:25.000000000 +0200
+++ Cheetah-1.0/SetupTools.py	2006-04-16 21:10:05.000000000 +0200
@@ -13,14 +13,14 @@
 ##################################################
 ## DEPENDENCIES ##
 
-from distutils.core import setup
+#from distutils.core import setup
 ## this bit is experimental:
-#try:
+try:
 #    # use http://peak.telecommunity.com/DevCenter/setuptools if it's installed
 #    # requires Py >=2.3
-#    from setuptools import setup
-#except ImportError:   
-#    from distutils.core import setup
+    from setuptools import setup
+except ImportError:   
+    from distutils.core import setup
 
 from distutils.core import Command
 from distutils.command.install_data import install_data
